src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs

```diff
@@ -776,7 +798,7 @@ async Task RequestCameraPermissions()
 			cameraPermissionsGranted = ContextCompat.CheckSelfPermission(Context, Manifest.Permission.Camera) == Permission.Granted;
 			if (!cameraPermissionsGranted)
 				permissionsToRequest.Add(Manifest.Permission.Camera);
-			if (Element.CaptureMode == CameraCaptureMode.Video)
+			if (Element?.CaptureMode == CameraCaptureMode.Video)
```

Comment:
There's no way to the element be null and the associeated renderer don't, right?

Reply:
The problem is that `Element` is not initialized when the class is instantiated:
```csharp
public CameraView? Element { get; set; }
```

Therefore, we have to make it nullable and assume that there is always a chance it could be null.

Reply:
There's a lot of strange cases like this in Xamarin.Forms because the framework forces us to initialize our Custom Renderers outside of the constructor in methods like `OnPropertyChanged`